reascripts/ReaSpeech/source/ReaSpeechWorker.lua

```diff
@@ -96,9 +96,9 @@ function ReaSpeechWorker:status()
   if self.active_job then
     local active_job = self.active_job
     if active_job.initial_request and not active_job.initial_request:ready() then
-      return 'Uploading'
+      return 'Sending Media'
```

Comment:
i'm not sure we're going to find verbiage that won't freak *someone* out lol

Reply:
Yeah, I had the same thought. :)

Reply:
maybe we should lean heavily in the other direction? say, for example, "Stealing Corporate Secrets" so that when they find out that no corporate secrets are actually being stolen that they'll be extra happy and relieved 🤔